pkg/machine/wsl/machine.go

```diff
@@ -95,7 +94,21 @@ func provisionWSLDist(name string, imagePath string, prompt string) (string, err
 
 	dist := env.WithPodmanPrefix(name)
 	fmt.Println(prompt)
-	if err = runCmdPassThrough("wsl", "--import", dist, distTarget, imagePath, "--version", "2"); err != nil {
+
+	// Run WSL import and capture output to check for known errors
+	// If virtual machine platform feature is not enabled it should fail with
+	// Error code: Wsl/Service/RegisterDistro/CreateVm/HCS/ERROR_NOT_SUPPORTED
```

Comment:
You should check more errors. For example I got `Wsl/Service/RegisterDistro/CreateVm/HCS/HCS_E_SERVICE_NOT_AVAILABLE` (when all features are turned off).

Reply:
You may want to update the comment here